tests/pytest/eligibility/test_views.py

```diff
@@ -71,6 +72,39 @@ def model_EnrollmentFlow_with_form_class(mocker, model_EnrollmentFlow):
     return model_EnrollmentFlow
 
 
+@pytest.mark.django_db
+def test_eligibility_logged_in_filtering_flows(mocker, model_TransitAgency, client):
```

Comment:
I really like that this single test covers that 1) digital flows are filtered "in" and 2) in-person flows are filtered out, as opposed to having two separate tests as you had pointed out in https://github.com/cal-itp/benefits/pull/2399#discussion_r1777666206. Nice job!

I think there is some code clean-up that can be done in this test. Some lines are redundant; more details are below in my comments on specific lines.